requirements.txt

```diff
@@ -1,14 +1,13 @@
-opera==0.6.6
-# enable building on alpine by avoiding rust
-cryptography==3.3.2
+# Package building dependecies
+wheel==0.37.1
+setuptools>=61.2.0
+setuptools_scm==6.4.2
```

Comment:
`setuptools_scm==6.4.2` does not match exactly with the version in `setup.cfg`, where it is `>=6.4.2` instead.

Reply:
Oops, I have updated this!